ui/elements/IconGroup/IconGroup.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import { StyledIconGroup } from './IconGroup.styles';
+import { IconGroupProps } from './types';
+import Icon from '../Icon/Icon';
```

Comment:
we use eslint and prettier to manage code consistency and formatting, to use these effectively, pls ensure that these extensions are installed in your code-editor:

1. https://marketplace.visualstudio.com/items/?itemName=dbaeumer.vscode-eslint
2. https://marketplace.visualstudio.com/items/?itemName=esbenp.prettier-vscode
3. https://marketplace.visualstudio.com/items/?itemName=rvest.vs-code-prettier-eslint

you can also configure format on save to format automatically by saving a file.

Reply:
currently, these files are not formatted correctly, and with eslint extension enabled we can see this highlighted 
<img width="942" alt="image" src="https://github.com/user-attachments/assets/da85996d-bb2a-4f41-bbef-cfde12862894" />